src/runTests.js

```diff
@@ -0,0 +1,88 @@
+// Command to run iOS or Android tests
+// `cavy run-ios` or `cavy run-android`
+//
+const server = require('../server');
+const { existsSync } = require('fs');
+const { spawn, execFileSync, execSync } = require('child_process');
+
+// If file changes have been made, revert them.
+function _teardown() {
+  console.log('cavy: putting your index.js back');
+  const cmd = 'mv index.js index.test.js && mv index.notest.js index.js';
+  console.log(`cavy: running \`${cmd}\`...`);
+  execSync(cmd);
+}
+
+// Borrowed from react-native-cli
+function _getAdbPath() {
+  return process.env.ANDROID_HOME
+    ? process.env.ANDROID_HOME + '/platform-tools/adb'
```

Comment:
Hmm we already have forward slashes here; so my point about the `./` is perhaps moot. Is this really from `react-native-cli`?! I wonder if they've improved this.

(Side note: We should probably test cavy-cli on Windows)

Reply:
Still the same in react-native-cli, although now they use template literals haha:
```
function getAdbPath() {
  return process.env.ANDROID_HOME
    ? `${process.env.ANDROID_HOME}/platform-tools/adb`
    : 'adb';
}
```